client/web/web_test.go

```diff
@@ -713,6 +713,61 @@ func TestServeAuth(t *testing.T) {
 	}
 }
 
+func TestNoOffSiteRedirect(t *testing.T) {
```

Comment:
I dont' think we need much of this setup.  If you're just testing a single page load and looking for the redirect, I don't think most of these other things are invoked (mocked localapi, etc)

Reply:
Yeah, wrote a new version using `NewServer`; much simpler and seems to work.